source/guides/account/scams-and-security.rst

```diff
@@ -0,0 +1,104 @@
+.. post:: 24 Feb, 2022
+    :category: Account
+    :tags: wallet
+    :excerpt: 1
+    :nocomments:
+
+######################
+Common Hacks and Scams
+######################
+
+***********
+Key Points
```

Comment:
This is kind of a special section. Wouldn't it be cool to highlight it somehow? Why don't you try to play with:

```rst
.. topic:: Key Points

   - **Be extremely cautious**...
```

Just a suggestion! Not mandatory at all.